scripts/docker/run-local.sh

```diff
@@ -6,5 +6,15 @@ else
     host_mountpoint=gd-fs
 fi
 
+host_mountpoint_parent=$(df -h --output=target "$host_mountpoint" | tail -n1)
+host_mountpoint_sharing=$(findmnt -n -o PROPAGATION "$host_mountpoint_parent")
+
+if [ "$host_mountpoint_sharing" != shared ]; then
+
+    docker run -i --privileged=true --entrypoint chroot \
+        --rm -v /:/host-root gooddata-fs \
+        /host-root/ nsenter -t 1 -m mount --make-shared "$host_mountpoint_parent"
```

Comment:
It's particularly interesting, that chroot seems to escape from the namespaces. I could see the host mounts in the chroot. However, I was not able to set the mountpoint propagation directly from the chroot without explicitly entering the PID 1 mnt namespace. PID 1 was already referring to the host init process..